### PR TITLE
converting newChannel to newChannel2 within gaia

### PR DESCRIPTION
**Pull request by ckerschb**
See Bug 1166948.

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2bf8af065843b6407b4649157f26b568047dbce9

File: build/download-manager.js [hunk 1]
```diff
@@ -98,7 +98,15 @@ var DownloadManager = {
                    .createInstance(Ci.nsIDownloader);
     downloader.init(listener, tmpFile);
 
-    let channel = Services.io.newChannel(url, null, null);
+    let channel = Services.io.newChannel2(url,
+                                          null,
+                                          null,
+                                          null,      // aLoadingNode
+                                          Services.scriptSecurityManager.getSystemPrincipal(),
```
Comment (KevinGrandon) [inline build/download-manager.js]: This line is throwing a JSHint error. You will need to fix the length, probably by changing the indentation.

build/download-manager.js: line 105, col 94, Line is too long. (ERROR)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7926555723195edca6222d2f379e55708e1f52af

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f03fb09c17fdf1c7e3c95f456ae5aa8d4ffcaeea

**Pull request closed without merging**